modules/op-starting-a-task-run-pipeline-run-build-user.adoc

```diff
@@ -70,17 +70,23 @@ spec:
   tasks:
   - name: fetch-repository # <1>
     taskRef:
-      name: git-clone
-      kind: ClusterTask
+      resolver: cluster
+      params:
+      - name: kind
+        value: task
+      - name: name
+        value: git-clone
+      - name: namespace
+        value: openshift-pipelines
+      - name: URL
+        value: $(params.URL)
```

Comment:
This one needs to be move to `tasks[].params` instead of under `tasks[].taskRef.params`

Reply:
thanks, fixed